core/variant/variant_op.h

```diff
@@ -775,6 +775,7 @@ class OperatorEvaluatorAddArray {
 		r_valid = true;
 	}
 	static inline void validated_evaluate(const Variant *left, const Variant *right, Variant *r_ret) {
+		*r_ret = Array{};
```

Comment:
Just a nitpick because we don't use this initialization style anywhere in the codebase.

```suggestion
		*r_ret = Array();
```

Reply:
Done :+1: